Birthday-2024-Project/ScenePrefabs/SettingsWindow.tscn

```diff
@@ -1,73 +1,109 @@
-[gd_scene load_steps=6 format=3 uid="uid://btcp1sgcebsyi"]
+[gd_scene load_steps=11 format=3 uid="uid://btcp1sgcebsyi"]
 
 [ext_resource type="Script" path="res://Scripts/UI/Settings/SettingsWindow.gd" id="1_ccik3"]
```

Comment:
>Repeat of a previous conversation

Was using Button_Hover instead because Button_Normal since button text was hard to see with Button_Normal's textured background. Edited Button_Normal to have solid white background like Button_Hover and set back Button_Normal for normal state